Client/Frontend/Home/HistoryPanel.swift

```diff
@@ -23,52 +23,64 @@ private typealias CategoryNumber = Int
 private typealias CategorySpec = (section: SectionNumber?, rows: Int, offset: Int)
 
 private struct HistoryPanelUX {
-    static let WelcomeScreenPadding: CGFloat = 15
-    static let WelcomeScreenItemTextColor = UIColor.grayColor()
-    static let WelcomeScreenItemWidth = 170
+    private static let WelcomeScreenItemTextColor = UIColor.grayColor()
+    private static let WelcomeScreenItemWidth = 170
+    private static let SyncedTabsCellChevronInset: CGFloat = 20
+    private static let SyncedTabsCellChevronSize: CGFloat = 20
+    private static let SyncedTabsCellImageSize: CGFloat = 32
+    private static let SyncedTabsCellHeight: CGFloat = 60
+    private static let SyncedTabsCellChevronLineWidth: CGFloat = 4.0
+    private static let SyncedTabsCellTextColor = UIColor(red: 92/255, green: 92/255, blue: 92/255, alpha: 1.0)
 }
 
-class HistoryPanel: SiteTableViewController, HomePanel {
+class HistoryPanel: UIViewController, HomePanel {
     weak var homePanelDelegate: HomePanelDelegate? = nil
-    private lazy var emptyStateOverlayView: UIView = self.createEmptyStateOverview()
-
-    private let QueryLimit = 100
-    private let NumSections = 4
-    private let Today = getDate(dayOffset: 0)
-    private let Yesterday = getDate(dayOffset: -1)
-    private let ThisWeek = getDate(dayOffset: -7)
-
-    // Category number (index) -> (UI section, row count, cursor offset).
-    private var categories: [CategorySpec] = [CategorySpec]()
-
-    // Reverse lookup from UI section to data category.
-    private var sectionLookup = [SectionNumber: CategoryNumber]()
-
-    var refreshControl: UIRefreshControl?
+    var profile: Profile!
```

Comment:
Nit: Is there a way we can replace this with non-explicitly unwrapped optional? Maybe pass in the profile in `init`? I'd like to see us move away from using these if we can.